src/components/atoms/Input/index.tsx

```diff
@@ -36,7 +38,9 @@ const Input: React.FC<InputProps> = ({
         {
           'border-gray-600': errorIcon === false,
           'border-scrummyRed-500': errorIcon === true,
-          'mb-10': errorIcon,
+          'mb-[px]': errorIcon && errorPadding === 'small',
+          'mb-[54px]': errorIcon && errorPadding === 'normal',
+          'mb-[66px]': errorIcon && errorPadding === 'large',
```

Comment:
Why we're using mb-[66px] instead of mb-16 which is 64px? Any particular reason?